torch/csrc/jit/pickler.cpp

```diff
@@ -714,17 +714,10 @@ OpCode Unpickler::readInstruction() {
               tensor_table_->at(data->elements().at(0).toInt()));
           break;
         case PicklerClass::INTLIST:
-          stack_.emplace_back(data->elements().at(0).toIntList());
-          break;
         case PicklerClass::TENSORLIST:
-          stack_.emplace_back(data->elements().at(0).toTensorList());
-          break;
         case PicklerClass::DOUBLELIST:
-          stack_.emplace_back(data->elements().at(0).toDoubleList());
-          break;
         case PicklerClass::BOOLLIST:
-          stack_.emplace_back(data->elements().at(0).toBoolList());
-          break;
+          stack_.emplace_back(data);
```

Comment:
Shouldn't this unwrap it from the tuple so these don't get returned as 1-element tuples? i.e. `emplace_back(data->elements().at(0))`